.github/workflows/map-diff.yml

```diff
@@ -0,0 +1,88 @@
+name: Map Diff
+
+on:
+  pull_request:
+    paths:
+    - '**.map'
+
+jobs:
+  comment-map-diff:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v3.0.0
+    - uses: actions/setup-node@v3.0.0
+      with:
+        node-version: '16'
+    - name: Package install
+      run: |
+        cd ./utils/wesnoth-map-diff
+        npm install
+    - name: Package build
+      run: |
+        cd ./utils/wesnoth-map-diff
+        npm run build:prod
+    - name: Get maps
+      id: get-maps
+      run: |
+        cd ./utils/wesnoth-map-diff
+
+        git fetch --depth=1 origin ${{ github.event.pull_request.base.sha }}
+
+        MAP_PATHS=$(git diff --name-status HEAD ${{ github.event.pull_request.base.sha }} | grep ".map$" | perl -pe 's/\w\s+//')
+        flat_map_paths=${MAP_PATHS[@]}
+        echo "::set-output name=MAP_PATHS::$flat_map_paths"
```

Comment:
Is it useful to split the work into multiple steps that are just more shell code? It seems to provide no benefit but require this magic to hand data over to the next step.

Without this you could handle the filepaths properly with an array instead.